kirovy/urls.py

```diff
@@ -18,12 +18,32 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from django.db import connection
 from drf_spectacular.views import SpectacularAPIView, SpectacularSwaggerView, SpectacularRedocView
 
 from kirovy.models import CncGame
 from kirovy.views import test, cnc_map_views, permission_views, admin_views, map_upload_views
 from kirovy import typing as t, constants
 
+def _get_games_url_patterns() -> list[path]:
+    """Return games related url patterns
+
+    I added this because games are retried from database, and it would crash the app
+    when migrations are not run.
+    @author rohsyl aka wu-shaolin
```

Comment:
```suggestion
    """Return URLs compatible with legacy CnCNet clients.

    - URLs are loaded when the :mod:`kirovy.urls` module is loaded, which happens when Django starts.
    - Checking the game slugs requires migration ``0002`` to have been run.
    
    These conditions caused a crash when running migrations for the first time, so now we return
    nothing if we detect that migrations haven't been run yet.
    
    .. codeauthor:: rohsyl aka wu-shaolin
    
    :return:
        A list of URLs that are backwards compatible with MapDB 1.0. Returns an empty list if migrations
        haven't been run yet.
```

A bit wordy, but it'll help those unfamiliar with python.